modules/magniteAnalyticsAdapter.js

```diff
@@ -174,6 +174,11 @@ const sendAuctionEvent = (auctionId, trigger) => {
 const formatAuction = auction => {
   const auctionEvent = deepClone(auction);
 
+  // Gather dm web vitals if this is the first auction
+  if (auctionEvent.aNum === 1) {
```

Comment:
is this name set in stone, needed by analytics team? can we change it? I think "aNum" doesn't really explain itself. `auctionCount` was a already a good name.